README.md

```diff
@@ -67,65 +33,80 @@ A React component class that injects subscriptions and actions into your compone
 
 #### Remarks
 
-* It needs to be invoked two times. The first time with its arguments described above, and a second time, with the component: `connect(mapPropsToSubscriptions, mapFirebaseToProps, mergeProps)(MyComponent)`.
+* It needs to be invoked two times. The first time with its arguments described above, and a second time, with the component: `connect(mapFirebaseToProps)(MyComponent)`.
 
 * It does not modify the passed React component. It returns a new, connected component, that you should use instead.
 
 #### Examples
 
 > Runnable examples can be found in the [examples folder](examples/).
 
-##### Inject `firebase` and `todos`
+##### Inject `todos`
 
   > Note: The value of `todos` is analogous to https://my-firebase.firebaseio.com/todos.
 
 ```js
-function mapPropsToSubscriptions() {
-  return { todos: 'todos' }
+const mapFirebaseToProps = {
+  todos: 'todos'
 }
 
-export default connect(mapPropsToSubscriptions)(TodoApp)
+export default connect(mapFirebaseToProps)(TodoApp)
 ```
 
 #####  Inject `todos` and a function that adds a new todo (`addTodo`)
 
 ```js
-function mapPropsToSubscriptions() {
-  return { todos: 'todos' }
-}
+const mapFirebaseToProps = (props, ref) => ({
+  todos: 'todos',
+  addTodo: todo => ref('todos').push(todo)
+})
 
-function mapFirebaseToProps(firebase) {
-  return {
-    addTodo: function(todo) {
-      firebase.child('todos').push(todo)
-    }
-  }
-}
+export default connect(mapFirebaseToProps)(TodoApp)
+```
+
+#####  Inject `todos`, `completedTodos`, a function that completes a todo (`completeTodo`) and one that logs in
 
-export default connect(mapPropsToSubscriptions, mapFirebaseToProps)(TodoApp)
+```js
+const mapFirebaseToProps = (props, ref, { auth }) => ({
+  todos: 'todos',
+  completedTodos: {
+    path: 'todos',
+    orderByChild: 'completed',
+    equalTo: true
+  },
+  completeTodo = id => ref(`todos/${id}/completed`).set(true),
+  login: (email, password) => auth().signInWithEmailAndPassword(email, password)
+})
+
+export default connect(mapFirebaseToProps)(TodoApp)
 ```
 
-#####  Inject `todos`, `completedTodos` and a function that completes a todo (`completeTodo`)
+### `<Provider firebaseApp>`
+
+By default `connect()` will use the [default Firebase App](https://firebase.google.com/docs/reference/js/firebase.app). If you have multiple Firebase App references in your application you may use this to specify the Firebase App reference available to `connect()` calls in the component hierarchy below.
+
+If you *really* need to, you can manually pass `firebaseApp` as a prop to every `connect()`ed component, but we only recommend to do this for stubbing `firebaseApp` in unit tests, or in non-fully-React codebases. Normally, you should just use `<Provider>`.
+
+#### Props
+
+* `firebaseApp` (*[App](https://firebase.google.com/docs/reference/js/firebase.app.App)*): A Firebase App reference.
+* `children` (*ReactElement*) The root of your component hierarchy.
```

Comment:
Fixed in https://github.com/unfold/react-firebase/commit/4abe20198448fd1c27525ac0c24d0d9fdd97ebf7